Formula/abook.rb

```diff
@@ -13,9 +12,16 @@ class Abook < Formula
     sha256 "b078b7af5c5fca8c97e693b70a0700ab91d9bed44bdccbf037ed5eb800c32d7b" => :sierra
   end
 
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "gettext"
   depends_on "readline"
 
+  # https://github.com/Homebrew/legacy-homebrew/issues/27338#issuecomment-37136980
+  patch :DATA
```

Comment:
Wow, that's an old one. Could we carry it in Homebrew/formula-patches instead of inline? (Note that I'm not 100% clear as to our policy here, just it seems cleaner to me to do that.)

Reply:
There's a policy?

Reply:
🤷 I sorta expect there to be for things like this - individual maintainer discretion only goes so far. 🤔

Reply:
Ah, so one of those elusive "unwritten" policies then… `¯\_(ツ)_/¯`

Reply:
https://docs.brew.sh/Formula-Cookbook#patches doesn't really say when you'd use one of the other, just that it's possible to do either. So I'm not going to bikeshed this any further as does seem to be a matter of personal preference.

Reply:
Thanks for putting up with my nitpicking! 😓

Reply:
Actually, it'll be simpler just to use an inreplace for this.